src/components/common/StaffPayment.vue

```diff
@@ -83,27 +78,12 @@ export default class StaffPayment extends Vue {
         break
     }
 
-    // Set Fee Summary data
-    this.setFilingData({
```

Comment:
This was heavy-handed as it overwrote the global filing data. This was OK for Corrections but we need this to work for Alterations too, so I moved this code to the parent view.

Reply:
(The parent has higher level knowledge and can update the filing data accordingly. This makes more sense for Alterations as there are multiple conditions to consider there.)